content/general/concepts/void/void.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This entry should be linked out in the [general data types entry ](https://www.codecademy.com/resources/docs/general/data-types). 
Void is in the "Other Common Data Types" table.

Reply:
Great call! 🙏🏻